lambda/src/main/java/org/dcsa/conformance/lambda/ApiLambda.java

```diff
@@ -11,22 +11,24 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Objects;
-import lombok.extern.slf4j.Slf4j;
+import lombok.extern.log4j.Log4j2;
 import org.dcsa.conformance.sandbox.ConformanceSandbox;
 import org.dcsa.conformance.sandbox.ConformanceWebRequest;
 import org.dcsa.conformance.sandbox.ConformanceWebResponse;
 import org.dcsa.conformance.sandbox.state.ConformancePersistenceProvider;
+import software.amazon.lambda.powertools.logging.Logging;
 
-@Slf4j
+@Log4j2
 public class ApiLambda
     implements RequestHandler<APIGatewayProxyRequestEvent, APIGatewayProxyResponseEvent> {
 
+  @Logging
   public APIGatewayProxyResponseEvent handleRequest(
       final APIGatewayProxyRequestEvent event, final Context context) {
     try {
-      System.out.println("event = " + event + ", context = " + context);
+      log.debug("event = {}; context = {}", event, context);
       JsonNode jsonEvent = OBJECT_MAPPER.valueToTree(event);
-      log.info("jsonEvent = " + jsonEvent.toPrettyString());
+      log.debug("jsonEvent = {}", jsonEvent::toPrettyString);
```

Comment:
Thanks for your review @gj0dcsa. I will move these debug logging back to info level, on all 3 Lambda entry point classes, since you need it for troubleshooting. I didn't expect that at this level, the input is required in the log. I'll test it in AWS before merging.